pandas/core/arrays/base.py

```diff
@@ -351,6 +351,17 @@ def __iter__(self):
         for i in range(len(self)):
             yield self[i]
 
+    def __contains__(self, item) -> bool:
+        """
+        Return for `item in self`.
+        """
+        # comparisons of any item to pd.NA always return pd.NA, so e.g. "a" in [pd.NA]
+        # would raise a TypeError. The implementation below works around that.
+        if isna(item):
+            return False
```

Comment:
is this going to affect e.g. DatetimeArray that does contain a NaT?

Reply:
I made a mistake in OP. Can you please read it again. `NaT in arr` should apparantæly return True, if an Nat is in the array.

I'll update the PR.